### PR TITLE
fix(@desktop/chats): restore last opened chat during app launch

### PR DESCRIPTION
**Pull request by mprakhov**
### What does the PR do

Restore the last open chat for different sections.

1. Create a new QSettings group
2. Store the last opened chat by key/value, where the key is section and the value is chat
3. Remove the key with value if the community was removed
4. Remove the key with value if the community does not have any chat

Fixes: #8889

Depends on:
- status-im/nimqml#44
- status-im/dotherside#84 

### Affected areas

Chat selection

### Screenshot of functionality (including design for comparison)

https://user-images.githubusercontent.com/117639195/210069116-ac439ec7-7821-424a-8096-76ee6f78b8a5.mov

### CODE REVIEW

**Review by jrainville — APPROVED** [review]

**Comment by jrainville** [comment]
Can you also post the links to the DOtherSide and nimqml PRs? Desktop's master should always point to the master/develop branches of its vendors, so we won't merge this PR until those are merged, and the vendors updated

**Comment by mprakhov** [comment]
> Can you also post the links to the DOtherSide and nimqml PRs? Desktop's master should always point to the master/develop branches of its vendors, so we won't merge this PR until those are merged, and the vendors updated

Done

**Review by jrainville — APPROVED** [review]

**Pull request merged**
